int32 check_node
float32 Pressure
string mobot_mode
string vehicle_mode
